app/javascript/components/admin/manage_users/ManageUsersTable.jsx

```diff
@@ -4,29 +4,50 @@ import PropTypes from 'prop-types';
 import { useTranslation } from 'react-i18next';
 import ManageUserRow from './ManageUserRow';
 import SortBy from '../../shared_components/search/SortBy';
+import ManageUsersRowPlaceHolder from './ManageUsersRowPlaceHolder';
 
-export default function ManageUsersTable({ users }) {
+export default function ManageUsersTable({ users, isLoading }) {
   const { t } = useTranslation();
 
   return (
     <Table className="table-bordered border border-2 mb-0" hover responsive>
       <thead>
         <tr className="text-muted small">
-          <th className="fw-normal border-end-0">{ t('user.name') }<SortBy fieldName="name" /></th>
-          <th className="fw-normal border-0">{ t('user.email_address') }</th>
-          <th className="fw-normal border-0">{ t('user.profile.role') }<SortBy fieldName="roles.name" /></th>
+          <th className="fw-normal border-end-0">{t('user.name')}<SortBy fieldName="name" /></th>
+          <th className="fw-normal border-0">{t('user.email_address')}</th>
+          <th className="fw-normal border-0">{t('user.profile.role')}<SortBy fieldName="roles.name" /></th>
           <th className="border-start-0" aria-label="options" />
         </tr>
       </thead>
       <tbody className="border-top-0">
+        {
+          isLoading
+            ? (
+              // eslint-disable-next-line react/no-array-index-key
+              [...Array(10)].map((val, idx) => <ManageUsersRowPlaceHolder key={idx} />)
+            )
+            : (
+              users?.length
+                ? (
+                  users?.map((user) => <ManageUserRow key={user.id} user={user} />)
+                )
+                : (
+                  <tr>
+                    <td className="fw-bold" colSpan="6">
+                      {t('user.no_user_found')}
+                    </td>
+                  </tr>
+                )
+            )
+        }
         {users?.length
           ? (
             users?.map((user) => <ManageUserRow key={user.id} user={user} />)
           )
           : (
             <tr>
               <td className="fw-bold" colSpan="6">
-                { t('user.no_user_found') }
+                {t('user.no_user_found')}
```

Comment:
Don't you need to remove this section here?

Reply:
What should it display after rendering the placeholder and finding no user?

Reply:
The comment is for the whole section (lines 43-50). Isn't it just a duplicate of lines 30-40?